aloe/result.py

```diff
@@ -38,36 +38,60 @@
 # started, which is when the stream is passed in.
 TERMINAL = [None]
 
-try:
+try:    # pylint: disable=too-complex
```

Comment:
The "too complex" message is because the whole class definition is inside the try block. Try moving it out, e.g.

```python
class ColoramaTerminal(...):
    # ...

try:
    from blessings import Terminal as BaseTerminal
except ImportError:
    BaseTerminal = ColoramaTerminal

Reply:
Fixed :) I wanted to avoid pylint thinking I had an "invalid-name" but it seems it doesn't think that.